tests/functional/adapter/materialization/test_prepared_statements.py

```diff
@@ -63,6 +63,8 @@ def run_seed_with_prepared_statements(
         # run models
         results = run_dbt(["run"], expect_pass=True)
         assert len(results) == 1
+        assert results[0].adapter_response.get("query_id")
+        assert results[0].adapter_response.get("compiled_query")
```

Comment:
Please add a test which checks a value of `compiled_query`.

Reply:
```
create or replace view
E           memory.test16740384229140444727_test_prepared_statements.materialization
E         security definer
E         as
E           select * from memory.test16740384229140444727_test_prepared_statements.seed


```
Do you know how 16740384229140444727 is generated?

Reply:
you can get it using `project.adapter.config.credentials.schema` like here for example https://github.com/starburstdata/dbt-trino/blob/master/tests/functional/adapter/test_custom_schema.py#L81

Reply:
Thanks, i realized i can find the compiled code for manifest. 
Do you want me to remove the `compiled_query` response?

And also, I am adding this for the purpose of DBT test. But adapter response is not reflecting in the main code.

I will raise and issue and PR there, to see how it goes.. So this might take a while.

Reply:
https://github.com/dbt-labs/dbt-core/pull/6645